keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,162 @@
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+*Note* that this KEP does not target any release; SIG Docs follows a continuous release process for website content.
+
+## Summary
+
+Feature docs are not a place for vendor pitches. Nevertheless, SIG Docs has seen pull requests that are more or less a request to place advertising content on the Kubernetes website.
+
+Feature docs also contain content hosted in multiple places (“dual-sourced content”). A good practice for code project docs is to host single-sourced content only, and to provide links to other providers’ single-sourced content.
+This KEP defines a policy on documentation content, so that authors can judge what is appropriate to propose and so that PR approvers can make consistent, fair decisions during the review process.
+
+## Motivation
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io, in line with its [charter](https://github.com/kubernetes/community/blob/master/sig-docs/charter.md#scope), and also sets standards for website content. Prior to this KEP there were no clear guidelines or standards showing what documentation is acceptable.
+
+The Kubernetes documentation is currently a mix of both 1) documentation describing the Kubernetes open source project; and 2) content describing how to install or use Kubernetes on several third party Kubernetes offerings.
+
+Some third party content is necessary in order for Kubernetes to function. For example, Docker, networking policy (CNI plugins), Ingress controllers, and logging all require third party components.
+Pages like [Logging Using Elasticsearch and Kibana](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/) are highly specific to a third party offering and seem more like third party product documentation than Kubernetes open source documentation.
+
+The goal of this KEP is to reach and document a consensus on what types of third party content are appropriate for inclusion in Kubernetes documentation; standards for including third-party content; and to drive a consistent policy toward third party content inclusion.
+This KEP focuses on the following issues related to third party content inclusion:
+
+1. What third party content is appropriate for inclusion in the Kubernetes documentation?
+1. Does third party content in sections such as [Getting Started](https://kubernetes.io/docs/setup/) in the docs provide sufficient value to the reader that they should remain?
+1. Is there a list of content pages that are so focused on third party product usage that they should be removed or updated from the Kubernetes documentation?
+1. When should the Kubernetes documentation host third party documents that we are not the source/authority of?
+1. How does the Kubernetes project handle third party content that is not kept up to date or hosts?
+1. Can feature owners flag when third party content is *required*, as opposed to preferable or common?
```

Comment:
interested how this can get implemented mechanically.

Reply:
@neolit123 Same!

Reply:
Maybe through the process of this Kep we nail down a list?